xbmc/guilib/FFmpegImage.cpp

```diff
@@ -375,17 +375,17 @@ bool CFFmpegImage::Decode(unsigned char * const pixels, unsigned int width, unsi
 
 bool CFFmpegImage::DecodeFrame(AVFrame* frame, unsigned int width, unsigned int height, unsigned int pitch, unsigned char * const pixels)
 {
-  AVPicture* pictureRGB = static_cast<AVPicture*>(av_mallocz(sizeof(AVPicture)));
+  AVFrame* pictureRGB = static_cast<AVFrame*>(av_mallocz(sizeof(AVFrame)));
```

Comment:
This is slightly broken. sizeof(AVFrame) is not part of the ABI. Use av_frame_alloc() (if I remember the function name right).